public/style.css

```diff
@@ -1,8 +1,5 @@
 @import url('http://fonts.googleapis.com/css?family=Open+Sans:400,400italic,700,700italic');
 @import url('http://fonts.googleapis.com/css?family=Montserrat&text=ZOONIVERSE');
```

Comment:
You're right. I just grabbed it from the live project. They might be using it somewhere else. I'll remove it.